Bio/NaiveBayes.py

```diff
@@ -62,12 +63,13 @@ def __init__(self):
 def calculate(nb, observation, scale=False):
     """Calculate the logarithmic conditional probability for each class.
 
-        - nb          - A NaiveBayes classifier that has been trained.
-        - observation - A list representing the observed data.
-        - scale       - Boolean to indicate whether the probability should be scaled by ``P(observation)``.  By default, no scaling is done.
+    - nb          - A NaiveBayes classifier that has been trained.
+    - observation - A list representing the observed data.
+    - scale       - Boolean to indicate whether the probability should be
+            scaled by ``P(observation)``.  By default, no scaling is done.
```

Comment:
You need the final line ``scaled by...`` to align under the first ``s`` of the previous line's ``   - scale...`` for this to be properly formatted RST bullet points.